Makefile

```diff
@@ -56,8 +56,8 @@ RELEASE_TOOL := ${BUILD_BIN_PATH}/release-tool
 GOLANGCI_LINT := ${BUILD_BIN_PATH}/golangci-lint
 
 ifeq ($(shell bash -c '[[ `command -v git` && `git rev-parse --git-dir 2>/dev/null` ]] && echo true'), true)
-	COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
-	GIT_COMMIT := $(if $(shell git status --porcelain --untracked-files=no),"${COMMIT_NO}-dirty","${COMMIT_NO}")
+	COMMIT_NO := $(shell git rev-parse --short HEAD 2> /dev/null || true)
```

Comment:
any reason for using the short commit?

Reply:
No I just thought it would be enough. Do you prefer the long one?

Reply:
I changed it back to the long one.